src/UID2.Client/UID2Encryption.cs

```diff
@@ -21,6 +21,8 @@ internal static class UID2Encryption
     {
         public const int GCM_AUTHTAG_LENGTH = 16;
         public const int GCM_IV_LENGTH = 12;
+        public const int TOKEN_V2_LENGTH = 133;
+        public const int TOKEN_V3_MIN_LENGTH = 163;
```

Comment:
Should we add `TOKEN_V3_MAX_LENGTH` as well?

Reply:
I thought about it, but imo this is good enough, especially if we increase the UID2 token length in the future

If we had a extra long token that somehow had correct key IDs, etc, I think the SDK would only read in the lengths it is looking for anyway? 

I can still add a max length if you think it's necessary

Reply:
The length is V3-specific though, I don't think that's going to change.

I imagine for future token versions we have separate upper and lower bounds.

Reply:
I guess the ad token length depends on the raw UID2 length, though. Hm.

Reply:
@vishalegbert-ttd are you saying v3 supports raw UIDs of any length?

Reply:
It looks like it?

Reply:
So should I cap it to 163 - 164? I don't want to add more validation than necessary

Reply:
Discussed offline, SDK currently supports any length raw UID2 for v3/v4 tokens so let's not add a max length.